src/all/mangapluscreators/src/eu/kanade/tachiyomi/extension/all/mangapluscreators/MangaPlusCreatorsDto.kt

```diff
@@ -1,68 +1,51 @@
 package eu.kanade.tachiyomi.extension.all.mangapluscreators
 
 import eu.kanade.tachiyomi.source.model.SChapter
-import eu.kanade.tachiyomi.source.model.SManga
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 @Serializable
 data class MpcResponse(
```

Comment:
use regular classes instead of data classes. unless needed

Reply:
The first paragraph in page you linked describes the use of data classes. It provides almost no help in parsing JSON unless you're printing it out. It only bloats the binary size.

Reply:
I'll remove it. However, I don't feel convinced about the trade off of pragmatics. If it were really an issue, I'd say the binary sizes can be drastically improved through some static [citation needed] analysis (a la what facebook seems to be [doing](https://github.com/facebookincubator/dataclassgenerate?tab=readme-ov-file)) and vector assets (seems to be [preferred](https://developer.android.com/training/multiscreen/screendensities#vector-graphics) now; ofc would be nice to not have to resort to simple flat letter icons and have an inkscape pro help us out with svgs but that's a different thing)